admin_manual/installation/apps_management_installation.rst

```diff
@@ -71,34 +71,59 @@ The configuration example below shows how to add a second directory, called ``ap
    :emphasize-lines: 9-13
 
 After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
-To do so, follow these steps:
+If you want to identify as an example all non core / non shipped apps, use the following command:
+
+::
+
+  sudo -uwww-data ./occ app:list --shipped=false
+
+Follow these steps to manually move apps:
 
 #. :ref:`Enable maintenance mode <maintenance_commands_label>`.
 #. :ref:`Disable the apps <apps_commands_label>` that you want to move.
-#. Create a new apps directory and assign it the same user and group, and ownership permissions as the core apps directory.
+#. Create a new apps directory like ``apps-external`` and assign it the same user, group and ownership permissions as the core apps directory.
 #. Move the apps from the old apps directory to the new apps directory.
-#. Add a new app directory in ``config/config.php``.
+#. Add the key for the new app directory in ``config/config.php``.
 #. If you're using a cache, such as `Redis <../configuration/server/caching_configuration.html#clearing-the-redis-cache>`_ or `Memcached <../configuration/server/caching_configuration.html#clearing-the-memcached-cache>`_, ensure that you clear the cache.
 #. Re-enable the apps.
 #. Disable maintenance mode.
 
 Script for moving non core apps in custom app directories
 ---------------------------------------------------------
 
-If you want to move your non core / non shipped apps in a custom app directory, to keep your installation clean, here is a script you can use.
+In the same way as you can manually move eg non core / non shipped apps to a custom directory, a script
```

Comment:
eg?

Reply:
e.g.

Reply:
will improve, pls let me know all your changes and I will do them in one shot

Reply:
Everything else was just wording changes as far as I could see.

What's important for me is that the script is there and that the script was not changed.

The other changes I am fine with. Nothing else to add from my side.

Reply:
e.g. or e.g.,